facdb/sql/_qaqc.sql

```diff
@@ -1,3 +1,9 @@
+-- Create full table of previous data (all records and all geometries that exist)
+ALTER TABLE dcp_facilities_with_unmapped DROP COLUMN IF EXISTS wkb_geometry;
+DROP TABLE IF EXISTS prev_full;
+SELECT a.*, b.wkb_geometry INTO prev_full
```

Comment:
The source of`dcp_facilities` is the shapefile on open data.  I wouldn't expect this file to have a wkb_geometry column; rather, when read into postgres the geometry column should just be `geom`.  Another way of going about this is not using the shapefile at all, but the lat and long fields from the csv.  If there is a lat value then there is a geometry.

Reply:
Regarding the name `prev_full` will down the line debugging cod we wonder what `prev_full` is and have to trace it back?  Maybe we add `facdb_` somewhere in this name?

Reply:
Ok shoot yea the with unmapped data has a lat/long which makes this join unnecessary. I didn't think to check that when downloading the data, addressing this now